src/applications/proxy-rewrite/sass/style-consolidated.scss

```diff
@@ -38,48 +38,6 @@ $font-stack: "Source Sans Pro", "Helvetica Neue", "Helvetica", "Roboto", "Arial"
   font-display: swap;
 }
 
-@font-face {
```

Comment:
I ran the injected header/footer locally and I'm seeing quite a few style regressions with spacing and fonts.

Please review the PR template; it has a reminder to check injected header/footer locally when making any changes to the VA.gov header.

### Desktop megamenu
<img width="1008" alt="Screenshot 2024-06-12 at 12 40 34 PM" src="https://github.com/department-of-veterans-affairs/vets-website/assets/19175324/f93db170-c32e-48f0-9757-f9d767667522">

### Desktop header
<img width="1144" alt="Screenshot 2024-06-12 at 12 40 26 PM" src="https://github.com/department-of-veterans-affairs/vets-website/assets/19175324/8988c264-16ce-4761-a7d7-6c786958c698">

### Mobile header / megamenu
<img width="459" alt="Screenshot 2024-06-12 at 12 40 15 PM" src="https://github.com/department-of-veterans-affairs/vets-website/assets/19175324/74c0401a-abc8-4e6c-a15d-57d8937dd811">

### Mobile header
<img width="459" alt="Screenshot 2024-06-12 at 12 40 07 PM" src="https://github.com/department-of-veterans-affairs/vets-website/assets/19175324/a2c44346-1f1e-4dc8-a134-8fe4849e992a">

### Desktop footer
<img width="1026" alt="Screenshot 2024-06-12 at 12 43 25 PM" src="https://github.com/department-of-veterans-affairs/vets-website/assets/19175324/7e91be1f-5166-41f5-a503-a892bf677715">

Reply:
@randimays I am not seeing these issues locally. Have you done a yarn install?

Reply:
@powellkerry Yes. I did a yarn install and rebuilt and still saw the issues. I also deleted your branch locally and repulled it, ran the yarn install and rebuilt again; still seeing the style issues 🤔

Reply:
@powellkerry You're verifying the injected header, correct?

Reply:
@randimays - yes, we'll verify the injected header. Kerry is gonna be OOO for a couple days, so I'm going to try my best to pick things up. I may ping you with questions about the injected header as I go. Thanks for flagging the issue!

Reply:
Thanks @micahchiang!

Reply:
@randimays - I've attached some screenshots that I think address the issues you surfaced, I'd appreciate if you took a look at them, then update your local branch to verify.

<details><summary>Desktop header + megamenu </summary>

![image](https://github.com/department-of-veterans-affairs/vets-website/assets/15097156/15302734-35df-41e1-a33f-ac0abf360a67)

</details>

<details><summary>Mobile header buttons</summary>

![image](https://github.com/department-of-veterans-affairs/vets-website/assets/15097156/e810db8f-8c9c-4eaa-b96c-d6bcab847b10)

![image](https://github.com/department-of-veterans-affairs/vets-website/assets/15097156/5b9a9a46-77fa-4d4a-86a2-b2cdfbd58376)

![image](https://github.com/department-of-veterans-affairs/vets-website/assets/15097156/a083a96d-1e58-497e-9c02-bf4e7fdbf45a)

</details>

<details><summary>Footer</summary>

![image](https://github.com/department-of-veterans-affairs/vets-website/assets/15097156/79304c06-b69b-4418-935c-57063857a635)

![image](https://github.com/department-of-veterans-affairs/vets-website/assets/15097156/19e0a5df-7647-4dae-a0e4-974078e762d2)

</details>

Reply:
@randimays - Here's an updated pic that includes the dropdown arrow icon: 
![image](https://github.com/department-of-veterans-affairs/vets-website/assets/15097156/494c9cd1-200d-4cfb-a0bd-179a78a0b0dc)

It seemed to be an issue with different `img` URLs between formation and css-library. Not too clear yet on why the difference caused a regression, think it has something to do with S3, I'm going to do a little more digging. In any case, it's back for now :smile: 

I wasn't able to reproduce the missing `+` symbols you mentioned in the footer, they still load okay for me.